src/Framework/Framework/Resources/Scripts/postback/postbackCore.ts

```diff
@@ -101,7 +101,7 @@ export async function postbackCore(
                 return await processPostbackResponse(options, context, postedViewModel, initialState, response.result, response.response!);
             } catch (err: any) {
                 if (err instanceof DotvvmPostbackError) {
-                    throw err;
+                    throw new DotvvmPostbackError(err.reason);
```

Comment:
We are losing stack trace here.

Reply:
I don't know why I made the change, it could be some hack since the method is async. Rethrowing using `throw err` works even in Chromium, so I'm reverting the change. If I rediscover the hack sometime, I'll make sure to add a comment explaining what it does :)